website/project/views/__init__.py

```diff
@@ -1 +1 @@
-from . import contributor, log, node, register, tag, file, comment  # noqa
+from . import contributor, log, node, register, tag, file, comment, drafts  # noqa
```

Comment:
Try to keep imports sorted when possible.

(Also, note to self: `file` is the name of a builtin, and therefore a terrible name for a module. That is outside the scope of this PR, however)